docs/actions/sql.md

```diff
@@ -0,0 +1,121 @@
+
+!!! note "Logging results"
+    You can configure the maximum number of results to print in the execution report for all actions by using
+    configuration property `chutney.tasks.sql.nbLoggedRow` in your project.  
+    Input value `nbLoggedRow` will override the value set by configuration.
+
+# Usage
+
+=== "Inputs"
+
+    | Required | Name          | Type          | Default | Note                                              |
+    |:--------:|:--------------|:--------------|:-------:|:--------------------------------------------------|
+    |    *     | `target`      | String        |         |                                                   |
+    |    *     | `statements`  | List<String\> |         |                                                   |
+    |          | `nbLoggedRow` | Integer       |   30    | Maximum number of rows to log in execution report |
+
+=== "Outputs"
+
+    |          Name  | Type                       | Note                                                                   |
+    |---------------:|:---------------------------|:-----------------------------------------------------------------------|
+    | `recordResult` | List<[Records](#records)\> | Each item in this list contains the resulting records of one statement |
+
+### Example
+
+Here is an example based one the following table : 
+
+| ID  | TITLE                    |    YEAR    | RATING |         DIRECTOR |
+|:---:|:-------------------------|:----------:|:------:|-----------------:|
+|  1  | "Castle in the Sky"      | 1986-08-02 |   78   | "Hayao Miyazaki" |
+|  2  | "Grave of the Fireflies" | 1988-04-16 |   94   |  "Isao Takahata" |
+|  3  | "My Neighbor Totoro"     | 1988-04-16 |   86   | "Hayao Miyazaki" |
+
+=== "Kotlin"
+``` kotlin
+SqlTask(
+    target = "ghibli_movies_database",
+    statements = listOf(
+        "SELECT * FROM movies WHERE rating > 85"
+    ),
+    nbLoggedRow = 5,
+    outputs = mapOf(
+        "numberOfBest" to "recordResult.get(0).count()".spEL(), // (1)
+        "bestMoviesTitles" to "records.stream().map(row -> row.get(\"TITLE\").value).collect(T(java.util.stream.Collectors).toList())".spEL() // (2)
+    )
+)
+```
+
+1. Expected result is 2
+2. Expected result is ["Grave of the Fireflies", "My Neighbor Totoro"]
+
+# Records
+
+One `Records` contains results for **one statement** and provides methods to retrieve and search them.  
+Results are structured by [columns](#column) and [rows](#row).
+
+Records provides the following attributes and methods which you can use in SpEL :
+
+* `affectedRows`: Returns the number of affected rows. This is useful for **INSERT**, **UPDATE** or **DELETE** statements  
+-> `${#records.affectedRows}`
+
+* `count()`: Returns the number of results from a **SELECT** statement.  
+-> `${#records.count()}`
+
+* `headers`: Returns the list of columns names (List< String >)  
+-> `${#records.headers}`
+
+* `columns`: Returns a list of [Columns](#column) (List< Column >).  
```

Comment:
I couldn't find how to properly escape `<>` in a bullet list :/
might needs a look in the python md/html generator if someone is up for a contribution there :)